src/createStoreSelector.ts

```diff
@@ -0,0 +1,14 @@
+import { action } from 'mobx';
+import getRootStore from './getRootStore';
+
+let createStoreAction = action('createStore', function createStoreAction(
+    key: string,
+    initialState: any
+) {
+    getRootStore().set(key, initialState);
+});
+
+export default function createStoreSelector<T>(key: string, initialState: T): () => T {
```

Comment:
>createStoreSelector [](start = 24, length = 19)

I don't like `createStoreSelector` -- it sounds like you're creating a selector, when what you're actually doing is creating a *store* and returning a selector to it.  I'd rather keep calling it `createStore`.